CalcualtorLogic.js

```diff
@@ -26,6 +26,7 @@ calculatorLogic.divide();
 class Calculator {
     currentResult = 0;
     resultElement = null;
+    operator = this.currentResult;
```

Comment:
can you explain what is the role of `operator` property is?

Reply:
saving the value of clicked button

Reply:
can you explain more?

Reply:
Just for knowing which operator button would be clicked

Reply:
in this case, why did you set `this.currentResult;` to this property?

Reply:
previously even I was not sure is this the requested property 
and if yes is this place is its correct place or some where else like inside the method

not it is not !!

thanks a lot